stellar-core-prometheus-exporter/debian/stellar-core-prometheus-exporter.service

```diff
@@ -11,7 +11,6 @@ StandardOutput=syslog
 StandardError=syslog
 SyslogIdentifier=stellar-core-prometheus-exporter
 ExecStart=/usr/bin/stellar-core-prometheus-exporter.py
-ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
I had to remove this as -HUP stops python webserver. So "reload" not only did not reload anything but broke things.

I think it's not a big deal for exporter to be restarted, prometheus is meant to handle missed scrapes gracefully.